README.md

```diff
@@ -1,9 +1,9 @@
 # Release Information 
 
-- **Version**: 2.1.0![](./docs/res/icon-preview.svg)
+- **Version**: 2.2.0![](./docs/res/icon-preview.svg)
 - **Certified**: Yes
 - **Publisher**: Fortinet
-- **Compatible Version**: 7.6.1 and later
+- **Compatible Version**: 7.6.2 and later
```

Comment:
@cs-abhishek-shukla, 
Why this change! Please keep the compatible version as 7.6.1.